ui/apps/akasha/src/index.tsx

```diff
@@ -24,6 +26,15 @@ export const register: (opts: IntegrationRegistrationOptions) => IAppConfig = op
   title: 'Ethereum World',
   logo: { type: LogoTypeSource.ICON, value: 'appAkasha' },
   i18nNamespace: ['app-akasha-integration', 'ui-lib-feed'],
+  menuItems: {
+    route: rootRoute,
+    label: 'Social',
+    name: 'app-akasha-integration',
```

Comment:
This name should be equal to the name in the registry. Let's avoid redefining names.
In the app-loader when you fire that event, you can specify the app name from there.